docs/Edit_Page.md

```diff
@@ -0,0 +1,40 @@
+# Edit page
+## Link: 
```

Comment:
this subheading...should be removed? or is it half a thought ready to be complete?

Reply:
To a user, there is no thing called "Edit page". I think maybe you want to move this content into Compounds.md, and have two main sections..."View Mode" and "Edit Mode"

Reply:
The Link subheading has the link format to this part of the site.

One issue with moving this into Compounds.md is that later we will probably have pages that are not just compounds and so this is meant to be a generalized reference to editing any type of page

Reply:
OK, then let's restructure so there's a "pages" reference and a "compounds" reference. Compounds says that it's a specialization of Pages, and links to pages.